contributions/catalog/webapps/static/css/style.css

```diff
@@ -527,3 +551,70 @@ input.datadel-api {
 .hidetext { -webkit-text-security: disc; }
 .showtext { -webkit-text-security: none; }
 
+h1 {
+    font-size: 32px;
+    height: 36px;
+    font-weight: 800;
```

Comment:
Font weights for h1, h2, h3, etc., are `normal` according to the design. 800px is used only when the font in the design is ProximaNova-Extrabld. Please update this.

Reply:
Putting font-family:ProximaNova-Extrabld  -- doesnt show a difference. Can discuss this change.

Reply:
Discussed about font changes and shared details about ProximaNova-Extrabld being ProximaNovv + font-weight: 800.